src/main/resources/templates/fragments/not-found-error.html

```diff
@@ -1,9 +1,5 @@
 <div
     th:if="${session.get('errorModel') == null or session.get('errorModel').getError() == '404'}">
     <h1>Die gesuchte Seite wurde nicht gefunden.</h1>
-    <p>
-        Versuchen Sie
-        <a href="/search" class="link-primary">eine Prüfung zu suchen</a>
-        oder gehen Sie zurück zur Startseite.
-    </p>
+    <p>Versuchen Sie doch eine Prüfung zu suchen.</p>
```

Comment:
```suggestion
    <p>Versuchen Sie stattdessen eine Prüfung zu suchen. Alternativ können Sie uns über die Informationen auf der Kontaktseite erreichen.</p>
```
I'd probably reword that to be a little bit more formal. Imagen being a stressed out student searching for his exam results and getting a snarky error page :sweat_smile: